build.gradle

```diff
@@ -293,14 +296,17 @@ dependencies {
     compile group: 'io.github.openfeign.form', name: 'feign-form', version: '3.8.0'
     compile group: 'io.github.openfeign.form', name: 'feign-form-spring', version: '3.8.0'
 
+    implementation (group: 'org.springframework.boot', name: 'spring-boot-starter-web'){
```

Comment:
We may have added spring boot starter web dependency twice, should we get rid of one of these dependencies ?